swagger_parser/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: swagger_parser
 description: Package that generates REST clients and data classes from OpenApi definition file
-version: 1.6.1
+version: 1.6.4
```

Comment:
1.7.0

Reply:
Edited in https://github.com/Carapacik/swagger_parser/pull/90/commits/b4ff6e95e2ff5a67979bd72fd4951282b67df4ad